modules/aws-eks-teams/outputs.tf

```diff
@@ -1,41 +1,41 @@
   value = tomap({
-    for k, v in aws_iam_role.platform_team : k => v.arn
+    for k, v in aws_iam_role.application_team_iam_role : k => v.arn
   })
 }
 
-output "application_teams_iam_role_arn" {
-  description = "IAM role ARN for Teams"
+output "application_team_sa_irsa_name" {
+  description = "IAM role name for Application Team EKS Service Account (IRSA)"
   value = tomap({
-    for k, v in aws_iam_role.team_access : k => v.arn
+    for k, v in aws_iam_role.application_team_sa_irsa : k => v.name
   })
 }
 
-output "team_sa_irsa_iam_role" {
-  description = "IAM role name for Teams EKS Service Account (IRSA)"
+output "application_team_sa_irsa_arn" {
+  description = "IAM role ARN for Application Team EKS Service Account (IRSA)"
   value = tomap({
-    for k, v in aws_iam_role.team_sa_irsa : k => v.name
+    for k, v in aws_iam_role.application_team_sa_irsa : k => v.arn
   })
 }
 
-output "team_sa_irsa_iam_role_arn" {
-  description = "IAM role ARN for Teams EKS Service Account (IRSA)"
+output "application_team_configure_kubectl" {
+  description = "Configure kubectl for each Application Team: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
   value = tomap({
-    for k, v in aws_iam_role.team_sa_irsa : k => v.arn
+    for k, v in aws_iam_role.application_team_iam_role : k => "aws eks --region ${data.aws_region.current.id} update-kubeconfig --name ${data.aws_eks_cluster.eks_cluster.name}  --role-arn ${v.arn}"
   })
 }
 
-output "platform_teams_configure_kubectl" {
-  description = "Configure kubectl for each Platform Team: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
+output "platform_team_iam_role_arn" {
+  description = "IAM role ARN for Platform Team EKS Access"
   value = tomap({
-    for k, v in aws_iam_role.platform_team : k => "aws eks --region ${data.aws_region.current.id} update-kubeconfig --name ${data.aws_eks_cluster.eks_cluster.name}  --role-arn ${v.arn}"
+    for k, v in aws_iam_role.platform_team_iam_role : k => v.arn
   })
 }
 
-output "application_teams_configure_kubectl" {
-  description = "Configure kubectl for each Application Teams: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
+output "platform_team_configure_kubectl" {
+  description = "Configure kubectl for each Platform Team: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
   value = tomap({
-    for k, v in aws_iam_role.team_access : k => "aws eks --region ${data.aws_region.current.id} update-kubeconfig --name ${data.aws_eks_cluster.eks_cluster.name}  --role-arn ${v.arn}"
+    for k, v in aws_iam_role.platform_team_iam_role : k => "aws eks --region ${data.aws_region.current.id} update-kubeconfig --name ${data.aws_eks_cluster.eks_cluster.name}  --role-arn ${v.arn}"
```

Comment:
note: changing output names is considered breaking change.
If we believe the original output names are not clear enough or confusing, we should change it with clear callout on next release for customer to know that we did it and what they should change from their end.

Reply:
I agreed so roll back to previous version. But you need to make breaking change for this output in near future...
I think we need to big changes for teams module for my Question 1.